images/win/post-generation/VSConfiguration.ps1

```diff
@@ -1,4 +1,7 @@
 $vsInstallRoot = Get-VisualStudioPath
 $devEnvPath = "$vsInstallRoot\Common7\IDE\devenv.exe"
 
-cmd.exe /c "`"$devEnvPath`" /updateconfiguration"
\ No newline at end of file
+cmd.exe /c "`"$devEnvPath`" /updateconfiguration"
+
+# Step to avoid initial delay for choco scripts
+choco upgrade chocolatey
```

Comment:
Let's create the separate script for this command. It is not related to Visual Studio actually.

Reply:
Done. thank you